src/main/java/net/sf/jabref/logic/groups/ExplicitGroup.java

```diff
@@ -168,10 +168,6 @@ public String getTypeId() {
         return ExplicitGroup.ID;
     }
 
-    public int getNumEntries() {
```

Comment:
I tried it now with a bigger database and although the population of the group tree was not "instantaneously", the performance was acceptable. So I merge this in.